pkg/sentry/fs/dev/net_tun.go

```diff
@@ -115,6 +126,35 @@ func (n *netTunFileOperations) Ioctl(ctx context.Context, file *fs.File, io user
 		if err != nil {
 			return 0, err
 		}
+
+		if req.Name() == "" {
```

Comment:
Have a look at test/syscalls/linux/tuntap.cc and see if you can't add a test for this there.

Reply:
Added